metric/manager.go

```diff
@@ -0,0 +1,61 @@
+package metric
+
+import (
+	"log"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespace = ""
+	subsystem = "wit"
+)
+
+var (
+	reqCnt = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "requests_total",
+		Help:      "Counter of requests received into the system.",
+	}, []string{"method", "entity"})
```

Comment:
changed action label to method which will hold http verb GET, POST etc.  Action label has many possible values which will increase metric size.